src/assets/javascripts/_/index.ts

```diff
@@ -44,6 +47,7 @@ export type Flag =
   | "search.highlight"                 /* Search highlighting */
   | "search.share"                     /* Search sharing */
   | "search.suggest"                   /* Search suggestions */
+  | "toc.follow"                       /* Following table of contents */
```

Comment:
Duplicate of below

Reply:
Probably should remove the one below instead to reduce merge conflicts in the future